### ISSUE TITLE
Add base Wikitext templates to schemas that can be shared with the community

### ISSUE THREAD

**Issue by lozanaross** [OpenRefine/CommonsExtension#86]
During the user testing sessions in Sept / Oct, we discussed the possibility to save Wikitext templates already inside schema templates. So if a schema for e.g. Artworks is created - this could include not only SDC statements but also 'prefilled' minimal Artwork wikitext template. This will be most useful to new users who don't have extensive experience with Wikitext and just need a simple and quick solution that will enable them to upload their media files. 

@trnstlntk can this be included in the documentation part of the SDC extension work?

**Comment by wetneb** [comment]
As long as the wikitext is constant (i.e. it does not rely on values in some columns) then this is already supported by the existing template.

**Comment by lozanaross** [comment]
Yes, this is just a documentation detail that @trnstlntk noted in the field notes from the user testing, and I'm recording it here so we don't forget, it's not a new feature. We can provide these ready-made schemas to download from a central documentation page.

**Comment by lokal-profil** [comment]
This issue should ideally have the #documentation label added to it

**Comment by lokal-profil** [comment]
> I realize that my comment above might be unclear. What I meant was that the current system of saved schemas can readily be used to provide pre-filled wikitext fields.
> 
> Because the predefined schemas are currently stored in the manifest, which is itself stored in OpenRefine, offering predefined schemas with a filled wikitext field would still involve making changes to OpenRefine itself (to update those schemas). But that would not involve more software development. But still not just documentation, I would say.

I see that [commons-manifest-v2.0.js](https://github.com/OpenRefine/OpenRefine/tree/master/extensions/wikibase/module/scripts/commons-manifest-v2.0.js) currently describes such predefined schemas (albeit without wikitext). E.g. ArtPhoto on [L397](https://github.com/OpenRefine/OpenRefine/blame/97bc585e47d77a86a33cc0fa8d30b6d1e8deef6a/extensions/wikibase/module/scripts/commons-manifest-v2.0.js#L397). But, on PAWS at least, those pre-defined schemas don't show up in the "Start from an existing schema" drop-down.

**Comment by wetneb** [comment]
Interesting! Then it would be worth checking if this also happens on a local install of OpenRefine and making an issue for it (or phabricator ticket if it only affects PAWS).
